src/App.tsx

```diff
@@ -1,61 +1,126 @@
+import { useState } from 'react';
 import './App.scss';
+import { User } from './types/User';
+import { Todos } from './types/Todos';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+const initialTodos: Todos[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+function getTodosId(todoId: Todos[]) {
+  const maxTodoId = Math.max(...todoId.map(todo => todo.id));
+
+  return maxTodoId + 1;
+}
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [isTitleError, setIsTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [isUserIdError, setIsUserIdError] = useState(false);
+
+  const [todoList, setTodoList] = useState<Todos[]>(initialTodos);
+  const getNewId = getTodosId(todoList);
```

Comment:
```suggestion
  const newId = getTodosId(todoList);
```
And use move this inside function where you use it